cves/kernel/CVE-2019-9503.yml

```diff
@@ -259,8 +275,10 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    Internationalization did not have any effect on this vulnerability. The
```

Comment:
Again, last sentence can be removed. Be specific as to why the vulnerability has nothing to do with i18n

Reply:
Updated